### PR TITLE
Post msg

### LINKED ISSUE #6: Allow posting a status :)
I'm on it.

screenshot at 15 58 22

### PR DESCRIPTION
**Pull request by n1k0**
Fixes #6. Deployed to gh-pages.

### CODE REVIEW

**Comment by n1k0** [comment]
It seems gh-pages are broken. The branch has been updated with the right code, but https://n1k0.github.io/tooty/ is not updated with it.

File: src/View.elm [hunk 1]
```diff
@@ -63,10 +63,90 @@ timelineView statuses label =
...
+                                ]
+                            ]
+                        , if hasSpoiler then
+                            div [ class "form-group" ]
+                                [ label [ for "spoiler" ] [ text "Spoiler" ]
+                                , textarea
+                                    [ id "spoiler"
+                                    , class "form-control"
+                                    , rows 5
+                                    , placeholder "This text will always be visible."
```
Comment (vjousse) [inline src/View.elm]: Hmm if it's a spoiler, it should not be always visible? :-)

Comment (n1k0) [inline src/View.elm]: It's the spoiler annoucement.

**Comment by n1k0** [comment]
spoiler

File: src/View.elm (hunk 1 above)
Comment (vjousse) [inline src/View.elm]: Oh boy, we need something clearer here. It took me some time to get it even after looking at your gif.
Maybe just renaming "Spoiler" to "Visible part" and "Status" to "Hidden part" or "Spoiler part", something along the lines?

Comment (n1k0) [inline src/View.elm]: Let's be frank, I was highly confused with the official client and I still do. No mention of followup convos:

screenshot at 20 18 40

**Comment by n1k0** [comment]
I'll follow up with more polish but I want to land things fast to avoid conflicts with you guys 😬

**Pull request merged**